src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx

```diff
@@ -48,6 +49,13 @@ export function SchemaTree(props: SchemaTreeProps) {
         dispatch(getSchemaAcl({path: activePath}));
     };
 
+    useEffect(() => {
+        // if the cached path is not in the current tree, show root
+        if (!currentPath.startsWith(rootPath)) {
+            handleActivePathUpdate(rootPath);
+        }
+    }, []);
```

Comment:
If it would be better to clean cache in cleanup function in useEffect? If we changed database - doesn't it mean that old caches are not useful any more?

Reply:
We may return to the same database, then it is ok to show the previously open node. With clearing on unmount it will always be root — and this is a change of the current behavior, which I don't think I want to do.